### PR TITLE
Add js primitive encodeURIComponent to globals.

### PR DESCRIPTION
**Pull request by fisx**
As suggested [here](https://github.com/anchor/servant-purescript/pull/3/files#r45406714).

### CODE REVIEW

File: src/Global.purs [hunk 1]
```diff
@@ -19,3 +19,6 @@ foreign import readInt :: Int -> String -> Number
 
 -- | Parse a floating point value from a `String`
 foreign import readFloat :: String -> Number
+
+-- | uri encoding
```
Comment (paf31) [inline src/Global.purs]: To be more specific, I'd say this "Encodes a URI _component_", since it escapes joining characters, unlike `escapeURI`.

**Comment by paf31** [comment]
Thanks! Could you please add the others as well though?

- `encodeURI`
- `decodeURI`
- `decodeURIComponent`

**Comment by fisx** [comment]
Done.  (I feel I should also update /docs, but I'm not sure how.  I'll look into it.)

**Comment by paf31** [comment]
Sorry, I didn't mean to close this, I think I hit the wrong button by mistake. I'll merge the other PR.

**Pull request closed without merging**